app/views/manageBusinesses/add/IncomeSourceAddedObligations.scala.html

```diff
@@ -97,105 +84,166 @@ <h1 class="govuk-panel__title">
     }
 }
 
+@getManageUrl = @{
+    controllers.manageBusinesses.routes.ManageYourBusinessesController.show(isAgent).url
+}
+
+@getNextUpdatesUrl = @{
+    if(isAgent) {
+        controllers.routes.NextUpdatesController.showAgent.url
+    } else {
+        controllers.routes.NextUpdatesController.show().url
+    }
+}
+
+@getReportingFrequencyUrl = @{
+    // TODO the below link will need to change to the new entry point for the opt in/out journeys once the page is made
```

Comment:
TODO is here to change the link when the new opt in/out entry point is developed.